bot/exts/help_channels/_message.py

```diff
@@ -92,6 +93,38 @@ async def is_empty(channel: discord.TextChannel) -> bool:
     return False
 
 
+async def dm_on_open(message: discord.Message) -> None:
+    """
+    DM claimant with a link to the claimed channel's first message, with a 100 letter preview of the message.
+
+    Does nothing if the user has DMs disabled.
+    """
+    embed = discord.Embed(
+        title="Help channel opened",
+        description=f"You claimed {message.channel.mention}.",
+        colour=bot.constants.Colours.bright_green,
+        timestamp=message.created_at,
+    )
+
+    embed.set_thumbnail(url=constants.Icons.green_questionmark)
+    embed.add_field(
+        name="Your message", value=truncate_message(message, limit=100), inline=False
+    )
+    embed.add_field(
+        name="Want to go there?",
```

Comment:
Let's formalise this a tad.
```suggestion
        name="Jump to conversation",
```

Reply:
So, the reason I chose this, is that both the field name and description will be "Jump to X", making it slightly repetitive. I tried something similar to what Sir Lancebot is using for `bookmark`.

Do you have any other suggestions of ways to formulate this, which are a bit more formal?

Reply:
Maybe just "Conversation" as the header and link being "Jump to first message"?

Reply:
That could definitely work. I'll update it.